src/web/editor.template.html

```diff
@@ -73,6 +73,7 @@
 </div>
 </div>
 <div id="footer"></div>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/es5-shim/4.5.7/es5-shim.min.js"></script>
```

Comment:
Nothing should break. I didn't realize we assumed ES5 compatibility, so I threw it in.